### PR TITLE
Throwing caught exceptions during JWT token refresh

### PR DESCRIPTION
**Pull request by prashant-shahi**
This PR resolves the issue (#71).
- When refreshing jwt token, throw caught exceptions.

*Note:*
**src/** - source files directory
**lib/** - generated files directory

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js/77)

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -46,17 +46,18 @@ export class DgraphClient {
         let payload: messages.Payload;
         const operation = async () => c.alter(op, metadata, options);
         try {
-            payload = await operation();
+            payload = types.createPayload(await operation());
```
Comment (paulftw) [inline src/client.ts]: The change to use `types.createPayload` seems unrelated to the fix, thus should be done in a separate PR.

Same below.

[the remark this answers is not in the document]
Comment (prashant-shahi) [reply src/client.ts]: I don't quite follow with your query.

[the remark this answers is not in the document]
Comment (BenKanouse) [reply src/client.ts]: I think the 'bot' is talking about line 50. The exception being caught has no type.

[the remark this answers is not in the document]
Comment (prashant-shahi) [reply src/client.ts]: The exception in `catch block` is of type `any`, and it doesn't require/accept a type definition.

I think I got it what they were referring to. The type definition of `payload` wasn't properly handled. Nice catch!

Thanks to @BenKanouse as well!

**Review by paulftw — APPROVED** [review]

File: src/client.ts (hunk 1 above)
Comment (prashant-shahi) [reply src/client.ts]: That's an older version. That change doesn't exist now.
